.github/workflows/release-branch.yml

```diff
@@ -27,21 +28,63 @@ jobs:
     - name: Create all-in-one installation script
       uses: ./.github/actions/gen-install-scripts
       with:
-        IMAGE_URL: ${{ secrets.DOCKER_RELEASE_REPO }}:${{ env.VERSION }}
-        ENV: default
+        IMAGE_URL: ${{ env.DOCKER_RELEASE_REPO }}:${{ env.VERSION }}
+        ENV: prod
 
     - name: Create branch and push it
       run: |
         git checkout -b "release/${VERSION}"
         git push origin "release/${VERSION}"
 
-    - name: Commit file and sign it
+    - name: Commit all-in-one
       uses: ./.github/actions/push-files
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         FILE_TO_COMMIT: "deploy/all-in-one.yaml"
         DESTINATION_BRANCH: "release/${{ env.VERSION }}"
 
+    - name: Commit crd project
```

Comment:
Splitting to steps is temporary - have created a ticket to address in in the future: CLOUDP-82782